raiden/raiden_event_handler.py

```diff
@@ -68,6 +70,28 @@
 )
 
 
+def unlock(
+        raiden: 'RaidenService',
+        payment_channel: PaymentChannel,
+        end_state: NettingChannelEndState,
+        participant: Address,
+        partner: Address,
+):
```

Comment:
add return None here. Also perhaps a small docstring to explain what this function does?